datasetIngestor/updateMetaData.go

```diff
@@ -3,60 +3,61 @@ package datasetIngestor
 import (
 	"encoding/json"
 	"io"
-	"log"
 	"net/http"
 	"strings"
 	"time"
-
-	"github.com/fatih/color"
 )
 
 const (
 	Classification = "classification"
-	AVLow = "AV=low"
-	AVMedium = "AV=medium"
-	INMedium = "IN=medium"
-	COLow = "CO=low"
+	AVLow          = "AV=low"
+	AVMedium       = "AV=medium"
+	INMedium       = "IN=medium"
+	COLow          = "CO=low"
 )
 
 /*
-getAVFromPolicy retrieves the AV (?) from a policy. 
+getAVFromPolicy retrieves the AV (?) from a policy.
 
 Parameters:
 - client: An HTTP client used to send requests.
 - APIServer: The URL of the API server.
 - user: A map containing user information. It should contain an "accessToken" key.
 - owner: The owner of the policy.
 
-The function constructs a URL using the APIServer, owner, and user's access token, and sends a GET request to this URL. 
-If the response status code is 200, it reads the response body and unmarshals it into a slice of Policy structs. 
-If there are no policies available for the owner, it logs a warning and sets the level to "low". 
+The function constructs a URL using the APIServer, owner, and user's access token, and sends a GET request to this URL.
+If the response status code is 200, it reads the response body and unmarshals it into a slice of Policy structs.
+If there are no policies available for the owner, it logs a warning and sets the level to "low".
 If there are policies available, it sets the level to the TapeRedundancy of the first policy.
 
 Returns:
 - level: The TapeRedundancy level of the first policy if available, otherwise "low".
 */
 func getAVFromPolicy(client *http.Client, APIServer string, user map[string]string, owner string) (level string) {
+	level = "low" // default value
+
 	var myurl = APIServer + "/Policies?filter=%7B%22where%22%3A%7B%22ownerGroup%22%3A%22" + owner + "%22%7D%7D&access_token=" + user["accessToken"]
-	resp, _ := client.Get(myurl)
+	resp, err := client.Get(myurl)
+	if err != nil {
+		return level
+	}
 	defer resp.Body.Close()
+	if resp.StatusCode != 200 {
+		return level
+	}
 
-	level = "low"
-	if resp.StatusCode == 200 {
-		body, _ := io.ReadAll(resp.Body)
-		type Policy struct {
-			TapeRedundancy string
-			AutoArchive    bool
-		}
-		var policies []Policy
-		_ = json.Unmarshal(body, &policies)
-		if len(policies) == 0 {
-			color.Set(color.FgYellow)
```

Comment:
should we keep the coloring?

Reply:
that seems like a block of code that was removed by this pr